server/interpreter.js

```diff
@@ -3923,6 +3995,72 @@ Interpreter.prototype.installTypes = function() {
     }
   };
 
+  /**
+   * Class for bound functions.  See ES5 §15.3.4.5 / ES6 §9.4.1.
+   * @constructor
+   * @extends {Interpreter.prototype.BoundFunction}
+   * @param {!Interpreter.prototype.Function} func Function to be bound.
+   * @param {Interpreter.Value} thisVal The this value passed into function.
+   * @param {!Array<Interpreter.Value>} args Arguments to prefix to the call.
+   * @param {?Interpreter.Owner=} owner Owner object (default: null).
+   */
+  intrp.BoundFunction = function(func, thisVal, args, owner) {
+    if (!func) return;  // Deserializing
+    intrp.Function.call(/** @type {?} */ (this), owner, func.proto);
+    /** @type {!Interpreter.prototype.Function} */
+    this.boundFunc = func;
+    /** @type {Interpreter.Value} */
+    this.thisVal = thisVal;
+    /** @type {!Array<Interpreter.Value>} */
+    this.args = args;
+    
```

Comment:
Blank line?

Reply:
There was probably some important code that I erased accidentally, but now I've covered my tracks by removing the `\n` as well.